sympy/ntheory/tests/test_digits.py

```diff
@@ -0,0 +1,39 @@
+import os
+import random
+import re
+
+random.seed(os.urandom(10 ** 6))
```

Comment:
Why not `random.seed(100)` or similar?

Reply:
Because `os.urandom` is more random than `random`, which is a pseudo RNG.